### ISSUE TITLE
After a long period of training, what will the training data accuracy incline?

### ISSUE THREAD

**Issue by yongyongjijip** [brilee/MuGo#26]
Will it finally get to 55% like alphago?

**Comment by brilee** [comment]
It's uncertain, because I haven't run the training as long as was cited in the paper. I'm dealing with various instabilities in the training process (methods with momentum, like adagrad or adam, seem to overfit somewhat)

**Comment by brilee** [comment]
I uploaded a model that gets 53% test accuracy
